CHANGELOG.md

```diff
@@ -5,6 +5,42 @@ All notable changes for each version of this project will be documented in this
 ## 12.1.0
 
 ### New Features
+- Added `IgxAccordion` component
+    - A collection of vertically collapsible igx-expansion panels that provide users with data and the ability to drill into it in a compact manner. The control is **not** data bound and takes a declarative approach, giving users more control over what is being rendered.
+    - Exposed two expansion modes, templating, easy-to-use keyboard navigation, option for nested accordions and more.
+    - Code example below:
+
+    ```html
+    <igx-accordion>
```

Comment:
This sample right here is way too big for the changelog. Let's make it as simple as possible. We can always add a little note stating that multiple expansion panels are supported.